### PR TITLE
feat: update deps and add more details to embed

### PR DESCRIPTION
**Pull request by castdrian**
- updates deps including /providers
- satisfies @thehairy's OCD
- shows season and episode that is being checked

### CODE REVIEW

**Review by binaryoverload — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -15,42 +15,43 @@
     "preinstall": "npx only-allow pnpm",
     "build": "tsc -b src && pnpm run copy",
     "copy": "cp src/tags.toml dist/tags.toml && cp src/mw-urls.txt dist/mw-urls.txt",
+    "copy": "cp src/tags.toml dist/tags.toml && cp src/mw-urls.txt dist/mw-urls.txt",
```
Comment (mrjvs) [inline package.json:18]: duplicate key?

**Review by mrjvs — CHANGES_REQUESTED** [review]
Looks good. Can we maybe possibly also add the "sent by <@...>" to tags. It should be a oneliner and its been on the todo list for a while?

File: package.json (hunk 1 above)
Comment (castdrian) [reply package.json:18]: huh lol, will get rid off, must be a weird rebase remnant or something

**Comment by castdrian** [comment]
> Looks good. Can we maybe possibly also add the "sent by <@...>" to tags. It should be a oneliner and its been on the todo list for a while?

will do in a min, /prov8ders has a new release anyway so can bump it up again as well

File: package.json (hunk 1 above)
Comment (castdrian) [reply package.json:18]: hm, not actually duplicate in the file itself, seems to be weird github diff nonsense

**Review by thehairy — CHANGES_REQUESTED** [review]
Whats with all the nonempty() replaces, was it removed?
imo nonempty() makes more sense than min(1)

**Comment by castdrian** [comment]
> Whats with all the nonempty() replaces, was it removed? imo nonempty() makes more sense than min(1)

deprecated

**Review by thehairy — APPROVED** [review]

**Review by binaryoverload — DISMISSED** [review]

**Review by mrjvs — APPROVED** [review]

**Pull request merged**